src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OverrideEqualsOnOverloadingOperatorEquals.Fixer.cs

```diff
@@ -27,10 +23,57 @@ public sealed override FixAllProvider GetFixAllProvider()
         }
 
         public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
```

Comment:
Well... not sre why it would be an IDE test.  Shouldn't it just be a roslyn-analyzers issue.  But sure, an issue is fine.